### PR TITLE
ENTESB-15633 - Main build on Github Actions

### PR DESCRIPTION
**Pull request by KurtStam**
https://issues.redhat.com/browse/ENTESB-15633

### CODE REVIEW

**Review by zregvart — APPROVED** [review]
LGTM

File: .github/workflows/on_pr.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: On PR
+
+on: 
+  pull_request
```
Comment (zregvart) [inline .github/workflows/on_pr.yml:4]: Actually, this can be
```suggestion
  pull_request_target
```

It's a bit safer that way.

**Comment by Delawen** [comment]
```There are test failures.```

Did we... mess up tests somewhere?

**Comment by KurtStam** [comment]
@Delawen I've disabled the test that is failing: https://issues.redhat.com/browse/ENTESB-15760 it failed both on GH and CircleCI

File: .github/workflows/on_pr.yml (hunk 1 above)
Comment (pakhuis2e) [reply .github/workflows/on_pr.yml:4]: Sounds good. I will do it in a follwup PR

Comment (KurtStam) [reply .github/workflows/on_pr.yml:4]: Sounds good - will do that in a followup PR or else it does not run with this PR.

**Pull request merged**